site/src/components/layout.js

```diff
@@ -4,22 +4,22 @@ import { Link } from 'gatsby'
 import Header from './header'
 
 function Layout({ location, title, children }) {
-  return (
-    <div className="main">
-      <header className="header-main">
-        <Header location={location} />
-      </header>
-      <main className="body-main">
-        {children}
-      </main>
-      <footer className="footer-main">
-        <div className="content">
-          Thanks! We hope you found what you are looking for. 
-          Please feel free to contribute via <a href="https://github.com/Morgan-Stanley/cpx-training">Github</a>.
+    return (
+        <div className="main">
+            <header className="header-main">
+                <Header location={location} />
+            </header>
+            <main className="body-main">
+                {children}
+            </main>
+            <footer className="footer-main">
+                <div className="content">
+                    Thanks! We hope you found what you are looking for.
+                    Please feel free to contribute via <Link href="https://github.com/Morgan-Stanley/cpx-training">Github</Link>.
```

Comment:
@mimiflynn It redirects for the foreseeable future (I have zero plans to change that) but we should probably play it safe and change the org to MorganStanley.

Reply:
Fixed!